python/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+cmake_minimum_required(VERSION 3.13.4)
+set(CMAKE_C_COMPILER clang)
+set(CMAKE_CXX_COMPILER clang++)
+
+project(pytilt)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_CXX_EXTENSIONS OFF)
+
+find_package(LLVM 15 REQUIRED CONFIG)
+add_definitions(${LLVM_DEFINITIONS})
+
+add_subdirectory(pybind11)
+
+add_library(tilt STATIC IMPORTED)
+set_target_properties(tilt PROPERTIES
+    IMPORTED_LOCATION ${TILT_DIRECTORY}/build/src/libtilt.so)
```

Comment:
Don't link libraries like this. You can add the library in cmake. Check how google test is imported

Reply:
Changed how tilt was linked.